packages/mediapipe-task-text/lib/src/tasks/text_classification/containers/text_classifier_result.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:ffi';
+import 'package:ffi/ffi.dart';
+import 'package:mediapipe_core/mediapipe_core.dart';
+import 'package:mediapipe_text/src/mediapipe_text_bindings.dart' as bindings;
+
+class TextClassifierResult {
+  const TextClassifierResult({
+    required this.classifications,
+    this.timestamp,
+  });
+
+  // The classification results for each head of the model.
+  final List<Classifications> classifications;
```

Comment:
Do we define Classifications? Or does MP? Uber nit but I would expect `Classification`, for which this is a `List<Classification> classifications`

Reply:
_I would have expected `Classification`, too_.